src/sidebar/test/app-controller-test.js

```diff
@@ -26,6 +27,8 @@ describe('AppController', function () {
   var fakeSettings = null;
   var fakeStreamer = null;
   var fakeWindow = null;
+  var fakeServiceConfig = null;
+  var fakeBridge = null;
```

Comment:
Can we keep this list sorted please? Incidentally, the `= null` here are purely a legacy of when this code was automatically converted from CoffeeScript. It doesn't serve any purpose.